### PR TITLE
[DOCS-457] Editorial scrub of "Building the Service Manager" doc.

### PR DESCRIPTION
**Pull request by rajkaramchedu**
@jimmyjames please review. No content change, only rewrites at several places to clean it up, including each new sentence on a new line.

### CODE REVIEW

File: cloud-and-lan-connected-device-types-developers-guide/building-cloud-connected-device-types/building-the-service-manager.rst [hunk 1]
```diff
@@ -68,15 +61,18 @@ There are two endpoints that all service manager SmartApps must define.
         path("/oauth/callback") {action: [GET: "callback"]}
     }
 
-The `/oauth/initialize` endpoint will be called during initialization and will ultimately be where we forward the user to the 3rd Party service so they can log in.
+The ``/oauth/initialize`` endpoint will be called during initialization.
+At this endpoint the SmartThings platform will forward the user to the third-party service so they can log in.
```
Comment (jimmyjames) [inline cloud-and-lan-connected-device-types-developers-guide/building-cloud-connected-device-types/building-the-service-manager.rst]: "At this endpoint" is probably not correct. Simply "This endpoint will forward the user ..."

File: cloud-and-lan-connected-device-types-developers-guide/building-cloud-connected-device-types/building-the-service-manager.rst [hunk 2]
```diff
@@ -85,52 +81,59 @@ authorization.
...
+- If we do not have a token from the third-party service, we start the OAuth flow by calling the SmartThings ``initialize`` endpoint. This is a static endpoint that will store a few bits of information about your SmartApp, such as the ``id``, and forwards the request to the ``/oauth/initalize`` endpoint defined in the SmartApp.
 
 Initialize Endpoint
 ~~~~~~~~~~~~~~~~~~~
 
-Used to initialize the OAuth flow to a 3rd party service. The initialize endpoint will save all of the query params passed to it, but requires three to be present. The SmartApp ID, the SmartApp's access token, and the api server url which is used to determine which server the SmartApp is installed on. The endpoint will then call the mapped `/oauth/initialize` endpoint defined in your SmartApp will all of the query params passed to it.
+This endpoint is used to initialize the OAuth flow to a third-party service. 
+The ``initialize`` endpoint will save all the query parameters passed to it, but requires below three to be present:
```
Comment (jimmyjames) [inline cloud-and-lan-connected-device-types-developers-guide/building-cloud-connected-device-types/building-the-service-manager.rst]: We use the full endpoint path elsewhere, we should be consistent so use ``/oauth/initialize`` here. Also "requires below three to be present" is confusing, maybe "requires the following three parameters:" or something like that

File: cloud-and-lan-connected-device-types-developers-guide/building-cloud-connected-device-types/building-the-service-manager.rst [hunk 3]
```diff
@@ -85,52 +81,59 @@ authorization.
...
+
+- The SmartApp ID, 
+- The SmartApp's access token, and 
+- The URL of the API that is used to determine which server the SmartApp is installed on. The endpoint will then call the mapped ``/oauth/initialize`` endpoint defined in the SmartApp with all the query parameters passed to it.
```
Comment (jimmyjames) [inline cloud-and-lan-connected-device-types-developers-guide/building-cloud-connected-device-types/building-the-service-manager.rst]: "The URL of the API that is used to determine ..." is misleading. Really it is just the installed URL of the SmartApp. Basically what happens is the `graph.api.smartthings.com/oauth/initialize` endpoint is called with the 3 specified params, then the platform forwards that request to the SmartApp's `/oauth/initialize` endpoint where the SmartApp will redirect to the 3rd party's site for login/authorization.

**Comment by rajkaramchedu** [comment]
@jimmyjames Made the changes, thanks! Please review.

**Pull request merged**